docs/source/getting_started/benchmarks/cifar10_benchmark.py

```diff
@@ -79,12 +93,18 @@
 path_to_train = '/datasets/cifar10/train/'
 path_to_test = '/datasets/cifar10/test/'
 
-# Use SimCLR augmentations, additionally, disable blur
+# Use SimCLR augmentations, additionally, disable blur for cifar10
 collate_fn = lightly.data.SimCLRCollateFunction(
     input_size=32,
     gaussian_blur=0.,
 )
 
+# Multi crop augmentation for SwAV
+swav_collate_fn = lightly.data.SwaVCollateFunction(
```

Comment:
We should change this to something like

```python
swav_collate_fn = lightly.data.SwaVCollateFunction(
    crop_sizes=[32],
    crop_counts=[2],
)
```
or just use the same collate function as for the other benchmarks in this case.